dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -668,4 +688,207 @@ private SourceCodeRepoInterface getSourceCodeRepoInterface(String gitUrl, User u
         Helper.checkEntry(result);
         return result.getVersions();
     }
+
+    @GET
```

Comment:
Create an issue to add a couple of JUnit tests for this function, at least one for CWL and for WDL.